packages/expo-av/package.json

```diff
@@ -31,22 +31,19 @@
   "author": "650 Industries, Inc.",
   "license": "MIT",
   "homepage": "https://docs.expo.io/versions/latest/sdk/av/",
-  "unimodulePeerDependencies": {
+  "peerDependencies": {
```

Comment:
I haven't been following the decisions closely but is this a change we've decided to move forward with in other modules?

Reply:
https://github.com/expo/expo/issues/8682 I think so ya